nx/include/switch/kernel/ipc.h

```diff
@@ -636,12 +636,12 @@ static inline Result ipcParseForDomain(IpcParsedCommand* r) {
         case DomainMessageType_Close:
             break;
         default:
-            return 0xF601;
+            return 0x1A60A;
```

Comment:
Change these to a new libnx error.

Reply:
I think this is supposed to match official sw, not sure.